libraries/tools/kotlin-annotation-processing/src/main/kotlin/org/jetbrains/kotlin/annotation/AnnotationProcessorWrapper.kt

```diff
@@ -81,16 +81,11 @@ public abstract class AnnotationProcessorWrapper(
             return
         }
 
-        val annotationsFile = processingEnv.getOptions().get(KAPT_ANNOTATION_OPTION)
-        if (annotationsFile != null) {
-            try {
-                kotlinAnnotationsProvider = FileKotlinAnnotationProvider(File(annotationsFile))
-            }
-            catch (e: IOException) {
-                kotlinAnnotationsProvider = EmptyKotlinAnnotationsProvider()
-            }
-        }
-        else kotlinAnnotationsProvider = EmptyKotlinAnnotationsProvider()
+        val annotationsFilePath = processingEnv.getOptions().get(KAPT_ANNOTATION_OPTION)
+        val annotationsFile = if (annotationsFilePath != null) File(annotationsFilePath) else null
+        kotlinAnnotationsProvider = if (annotationsFile != null && annotationsFile.exists()) {
+            FileKotlinAnnotationProvider(annotationsFile)
+        } else EmptyKotlinAnnotationsProvider()
```

Comment:
Could you please put `else` on the new line and add `{}` for it? So it would be agreed with our coding conventions.